### PR TITLE
Add sample commands to UI

### PR DESCRIPTION
**Pull request by csehlke**
#78

- [x] wait for #77 before merging
- [x] change target branch back to devlop

### CODE REVIEW

File: UI/src/app/home/home.component.css [hunk 1]
```diff
@@ -38,9 +38,8 @@
     margin: 250px auto 0 auto;
 }
 .display-container{
-    width: 840px;
+    width: 700px;
     height: 560px;
-    border: solid white 2px;
```
Comment (csehlke) [inline UI/src/app/home/home.component.css]: if you enlarge the container, you need to scroll to see the error message, I would have kept it like that

I also thought the box looked nice, so i kept it

File: UI/src/app/home/home.component.html [hunk 2]
```diff
@@ -30,6 +30,8 @@
     </div>
 
     <hr>
+	
+    <label *ngIf="columns && columns.length > 0">Available columns in Table</label>
```
Comment (csehlke) [inline UI/src/app/home/home.component.html]: anyone opposed to 'Available columns in **the t**able`

Comment (csehlke) [inline UI/src/app/home/home.component.html]: anyone opposed to 'Available columns in **the t**able`

File: UI/src/app/working-commands/working-commands.component.html [hunk 3]
```diff
@@ -5,19 +5,64 @@
...
+            <li>Draw a scatter plot of age against salary for “Active” employees</li>
+            <li>Draw age vs salary on a scatter plot</li>
+            <li>age vs salary on a scatter plot</li>
+            <li>average age vs maximum salary on a bar chart per department</li>
+            <li>Show number of employees vs maximum salary on a bar chart per department</li>
+            <li>Bar chart of min age per race</li>
+            <li>Pie chart of average pay rate per department</li>
         </ul>
         <h3>INDICATOR_DATASET</h3>
+        <h4>Synonyms for words like Plot(Make, Draw) etc should work the following queries also</h4>
```
Comment (csehlke) [inline UI/src/app/working-commands/working-commands.component.html]: maybe this should be added for the first data set as well

**Review by shayansiddiqui — APPROVED** [review]

File: UI/src/app/working-commands/working-commands.component.html (hunk 3 above)
Comment (csehlke) [reply UI/src/app/working-commands/working-commands.component.html]: @shayansiddiqui

**Comment by csehlke** [comment]
looks good

**Pull request merged**